trajopt/include/trajopt/kinematic_terms.hpp

```diff
@@ -13,14 +13,18 @@ namespace trajopt
 using namespace sco;
 typedef BasicArray<Var> VarArray;
 
-struct CartPoseErrCalculator : public TrajOptVectorOfVector
+/**
+ * @brief The CartPosErrCalculator Used to calculate the error for CartPosTermInfo
```

Comment:
I recommend removing `The CartPoseErrCalculator`

Reply:
I recommend this for the others.

Reply:
I agree. I had considered renaming it to DynamicCartPoseErrCalculator or something similar.  I'll remove the current CartPos and drop the Static on the other one.